pom.xml

```diff
@@ -50,11 +50,12 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <gradle.version>5.6</gradle.version>
+        <gradle.version>8.9</gradle.version>
         <gson.version>2.11.0</gson.version>
         <moshi.version>1.12.0</moshi.version>
         <jackson2x.version>2.17.2</jackson2x.version>
-        <jackson2x.databind.version>2.17.2</jackson2x.databind.version>
```

Comment:
This looks like an improvement.  We should probably be using the `jackson-bom` in this project, but we don't need to tackle that now.

Reply:
Sure, I can do it

Reply:
done